testbed/tests/brakelight_test.py

```diff
@@ -0,0 +1,46 @@
+# firmware/testbed/tests/brakelight_test.py
+
+import time
+from testbeds.hil_testbed import teststand
+from drivers.common_drivers.can_driver import HILBoard
+from tqdm import tqdm
+
+# Define constants for the threshold voltage when the brake light turns on
+BRAKE_LIGHT_ON_THRESHOLD_VOLTAGE_1 = 833 # Lower threshold voltage
+BRAKE_LIGHT_ON_THRESHOLD_VOLTAGE_2 = 841 # Upper threshold voltage
+
+def test_brakelight(teststand):
+    print(teststand.vehicle_boards)
+    
+    vcu_hil: HILBoard = teststand.hil_boards["vcu_hil"]
+    # update pud -> pdu_hil
+    pdu_hil: HILBoard = teststand.hil_boards["pdu_hil"]
+
+    # Iterate over the entire voltage range
+    for voltage in tqdm(range(0, 3301)):
+    
+        # Set the brake position voltage
+        vcu_hil.set_signal("BrakePosition", {"BrakePosition": voltage}) # Reference to HIL.dbc line 69
+        time.sleep(0.001)
```

Comment:
I believe this is used as a propagation delay to allow the HIL board to update the value. Was this tuned experimentally or is it something that we can abstract into a set of common helper functions for others to use in their tests too? At the very least I want this 0.001 to become a constant with a name to the effect of EXPERIMENTAL_BRAKE_POSITION_PROPAGATION_DELAY_S